### PR TITLE
chore: Remove Nuxt web-app and related code

### PR DESCRIPTION
**Pull request by McPizza0**
## What does this PR do?

Removes all of the previous NuxtJS version of UnInbox

## Type of change

EVERYTHING

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#551** <a href="https://app.graphite.dev/github/pr/un/inbox/551?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: package.json [hunk 1]
```diff
@@ -6,14 +6,13 @@
   "scripts": {
     "format": "prettier . --write",
     "check:format": "prettier . --check",
-    "check:types": "dotenv -e .env.local -- turbo run check --filter=!@u22n/web-app --cache-dir=.turbo",
+    "check:types": "dotenv -e .env.local -- turbo run check --filter=!@u22n/web --cache-dir=.turbo",
```
Comment (BlankParticle) [inline package.json:9]: remove the all the `--filter` entirely, they were there to prevent nuxt app form being checked

**Comment by McPizza0** [comment]
🫡

**Review by BlankParticle — APPROVED** [review]
Lets go

**Comment by McPizza0** [comment]
### Merge activity

* **Jul 10, 8:43 AM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/551).

**Pull request merged**